testsuite/FixedLocalTuple/TestFixedLocalTupleList.py

```diff
@@ -54,7 +57,7 @@ def setUp(self) :
         cellGrid       = espressopp.tools.decomp.cellGrid(box, nodeGrid, cutoff, system.skin)
         
         print 'NodeGrid = %s'%(nodeGrid,)
-        print 'CellGrid = %s'%cellGrid
+        print 'CellGrid = %s'%(cellGrid,)
```

Comment:
Why?

Reply:
Seems to be a bug with Int3D(), but now it is solved

Reply:
So still why?

Reply:
Well, the %s denotes a string like (1,2,3) which is separated by comma. the original problem was that the cellGrid was in some testCases returned as string and not as an Int3D. Nonentheless, for either case "pure string (1,2,3)" or "Int3D(1,2,3)" the current print command is valid.

Reply:
ok